docs/source/algorithms/LorentzCorrection-v1.rst

```diff
@@ -12,9 +12,9 @@ Description
 
 Calculates and applies the lorentz correction weights to a workspace. The Lorentz correction *L* is calculated according to:
 
-.. math:: 
-   L = \sin(\theta)^{2}/\lambda^{4}
-   
+.. math::
+   L = \frac{\sin^{2}\theta}{\lambda^{4}}
```

Comment:
Somewhere there needs to be specified that this is valid only for TOF diffractometers